cli/pcluster/pcluster.py

```diff
@@ -370,6 +372,31 @@ def stop(args):
         set_asg_limits(asg_name=asg_name, config=config, min=0, max=0, desired=0)
 
 
+def get_version(stack):
+    """
+    Get the version of the stack if tagged.
+
+    :param stack: stack object
+    :return: version or empty string
+    """
+    tags = filter(lambda x: x.get("Key") == "Version", stack.get("Tags"))
+    return tags[0].get("Value") if len(tags) > 0 else ""
```

Comment:
`Unexpected error of type TypeError: object of type 'filter' has no len()`
we have to use `len(list(tags))`

Reply:
fixed